integration-tests/kubernetes-client/src/main/resources/application.properties

```diff
@@ -1,3 +1,4 @@
 quarkus.kubernetes-config.enabled=true
 quarkus.kubernetes-config.config-maps=cmap1,cmap2
 quarkus.kubernetes-config.secrets=s1
+quarkus.kubernetes-config.namespace=test
```

Comment:
I am not sure I like the idea of having our main integration test use the less likely use case